app.py

```diff
@@ -11,17 +11,9 @@
     app.run()
 
 
-@app.route('/') # ADD 'UNIQUE' BEFORE 'USERNAME' IN TABLE CREATION
+@app.route('/')
 def hello_world():
-    conn = sqlite3.connect('data_base.db')
-    curs = conn.cursor()
-    curs.execute('''CREATE TABLE IF NOT EXISTS users ( 
-                       Id INTEGER PRIMARY KEY AUTOINCREMENT,
-                       Username TEXT NOT NULL UNIQUE,
-                       Password TEXT NOT NULL
-                       )''')
-    conn.commit()
-    conn.close()
+    db_management.create_connection()
```

Comment:
Why do you need this view? Do you want all of your tables to be created each time someone accesses the "/" path?

Also, you names this function "create_connection", but you don't create a connection, right? you create your tables here.